WesWang/00-README.md

```diff
@@ -1,3 +0,0 @@
-Hi, my name is Wesley Wang.
-Feel free to call me Wes.
-I look forward to mastering Python after this class.
```

Comment:
Thanks for removing this! I should have reviewed earlier PRs (not just blindly merging them) and pointed this out.